frontend/test/metabase/scenarios/admin/troubleshooting/help.cy.spec.js

```diff
@@ -35,8 +35,8 @@ describeOSS("scenarios > admin > troubleshooting > help", () => {
       .parents("a")
       .should("have.prop", "href")
       .and(
-        "equal",
-        "https://www.metabase.com/help?utm_source=in-product&utm_medium=troubleshooting&utm_campaign=help",
+        "match",
+        /^https:\/\/www\.metabase\.com\/help\?utm_source=in-product&utm_medium=troubleshooting&utm_campaign=help&instance_version=v0\.(?:(?!diag=).)+$/,
```

Comment:
`(?:xxxxx)` = non-capturing group i.e. what's after `?:` wouldn't be considered as a capturing group
`(?!xxxx)` = negative look ahead i.e. what's after `?!` can't exist

so, `(?:(?!diag=).)+` means `.` could be anything just don't contain any `diag=` after.